jobmon_gui/src/task_details_page/task_details.tsx

```diff
@@ -78,11 +78,12 @@ function TaskDetails() {
         return () => clearInterval(interval);
     }, [taskId, task_status]);
 
+    const navigate = useNavigate()
     return (
         <div>
             <Breadcrumb>
-                <Breadcrumb.Item><Link to="/">Home</Link></Breadcrumb.Item>
-                <Breadcrumb.Item><Link to={{ pathname: `/workflow/${workflow_id}/tasks` }}>Workflow ID {workflow_id}</Link></Breadcrumb.Item>
+                <Breadcrumb.Item><button className="breadcrumb-button" onClick={() => navigate(-2)}>Home</button></Breadcrumb.Item>
```

Comment:
what's the -2 represent? Taking a guess, I read it as "navigate 2 pages back" which maybe isn't ideal to hardcode in case we change the layout

Moot point if it represents something else

Reply:
You are correct, it does mean navigate two pages back. It's how ReactRouter recommends handling in this situation. If we start changing the layout we'll need to start keeping track of specific URLs which I'm fine with, but can handle it when it comes.